content/blog/2019/10/2019-10-21-thinking-about-jenkins-security.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Thinking About Jenkins Security - DevOps World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- security
+- event
+- community
+authors:
+- markewaite
+- wadeck
+- stackscribe
+opengraph:
```

Comment:
Add a `note` to indicate it is a speaker blogpost

Reply:
Resolved in e02460f75da1eded63e0fc1299ecfd53f93f6a8d